snmp/tests/utils.py

```diff
@@ -15,7 +15,7 @@
 @contextlib.contextmanager
 def mock_profiles_root(root):
```

Comment:
Let's rename to reflect the new behavior:

```suggestion
def mock_profiles_confd_root(root):
```